extensions/csv/src/main/java/io/deephaven/csv/tokenization/external/DoubleParser.java

```diff
@@ -0,0 +1,612 @@
+package io.deephaven.csv.tokenization.external;
+
+/*
+ * @(#)FastDoubleParser.java
+ * Copyright © 2021. Werner Randelshofer, Switzerland. MIT License.
+ */
```

Comment:
We likely need to move this to its own module and license as appropriate.

Reply:
Agree